src/vue/pages/VerificationGeneral.vue

```diff
@@ -270,8 +277,9 @@ export default {
         first_name: this.form.personal.firstName,
         last_name: this.form.personal.lastName,
         documents: {
-          [DOCUMENT_TYPES.kycAvatar]:
-            this.form.documents.avatar.getDetailsForSave(),
+          [DOCUMENT_TYPES.kycAvatar]: this.form.documents.avatar
+            ? this.form.documents.avatar.getDetailsForSave()
+            : EMPTY_DOCUMENT,
```

Comment:
why do we use the fallback value only for "avatar"? are other docs required? if yes, explain it in a comment pls

Reply:
Yes, other docs required